packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -253,6 +253,7 @@ class SnackBar extends StatelessWidget {
       accentColorBrightness: theme.accentColorBrightness,
       snackBarTheme: snackBarTheme,
     );
+    final TextStyle contentTextStyle = snackBarTheme.contentTextStyle ?? darkTheme.textTheme.subhead;
```

Comment:
Would it make sense to default to `darkTheme.textTheme.subhead` in regular theme, and `lightTheme.textTheme.subhead` in dark theme?

Reply:
It would but that is out of scope for the work I'm doing here, it is tracked here: https://github.com/flutter/flutter/issues/31418 and will be (hopefully) done soon after.

Additionally would need to consider what color the background of the snackbar is in that scenario.

Reply:
Yeah but I don't want to adjust the defaults yet as that will be a visually breaking change. It's tracked by https://github.com/flutter/flutter/issues/31418 and will come later down the road.

Reply:
Ack